install.md

```diff
@@ -94,6 +100,20 @@ Then to install Buildah on Ubuntu follow the steps in this example:
   git clone https://github.com/projectatomic/buildah ./src/github.com/projectatomic/buildah
   cd ./src/github.com/projectatomic/buildah
   PATH=/usr/lib/go-1.8/bin:$PATH make runc all TAGS="apparmor seccomp"
-  make install
+  sudo make install
   buildah --help
 ```
+
+## Debian
+
+To install the required dependencies, you can use those commands, tested under Debian GNU/Linux amd64 9.3 (stretch):
+
+```
+gpg --recv-keys 0x018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D
+gpg --export 0x018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D >> /usr/share/keyrings/projectatomic-ppa.gpg
+echo 'deb [signed-by=/usr/share/keyrings/projectatomic-ppa.gpg] http://ppa.launchpad.net/projectatomic/ppa/ubuntu zesty main' > /etc/apt/sources.list.d/projectatomic-ppa.list
```

Comment:
note that we only add a single PPA here - projectatomic's. Debian stretch already has golang 1.8 (not 1.8.1 mind you, in backports), but I wonder if the flatpak PPA is also necessary... The only thing I can see would be required is libostree-dev, but that's also available in Debian (2017.12-2~bpo9+1 in backports). It would be nice to know which version of ostree is needed here as well...

Reply:
@giuseppe PTAL